streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -23,7 +23,6 @@
 import kafka.utils.ZkUtils;
```

Comment:
Related to the comment below, we can probably remove this zkUtils object and the import.